site/src/component/CoursePopover/CoursePopover.tsx

```diff
@@ -43,6 +44,14 @@ const CoursePopover: FC<CoursePopoverProps> = ({ course, interactive = true, req
         <PrerequisiteText course={course} />
         <CorequisiteText course={course} />
         <IncompletePrerequisiteText requiredCourses={requiredCourses} />
+        {course.terms && course.terms.length > 0 && (
+          <div className="quarter-offerings">
+            <b>Quarter Offerings:</b>
+            <div className="quarter-indicator-wrapper">
+              <CourseQuarterIndicator terms={course.terms} size="sm" />
+            </div>
+          </div>
+        )}
```

Comment:
Take a look at `AddCoursePopup.tsx` lines 83-86. Would it be worth it to combine that with these lines in a separate component and just call that component in both places? That should also let the style be shared between the two.